docs/contributing/CONTRIBUTING.md

```diff
@@ -1,16 +1,27 @@
 # Contributing
 
+Welcome! So glad you've decided to help make Cursorless better. Once you've
+learned how to [install](#installation) and [run / test a local copy of the
+extension](#running--testing-extension-locally), you may want to check out the
+[API docs](api) to learn more about how Cursorless works.
+
 ### Installation
 
 ```sh
-yarn install
+yarn
 ```
 
-### Running tests
+### Running / testing extension locally
 
-```sh
-yarn run test
-```
+In order to test out your local version of the extension or to run unit tests
+locally you need to run the extension in debug mode. To do so you need to run
+the `workbench.action.debug.selectandstart` command and then select either "Run
```

Comment:
Super helpful. Realized I was running the tests incorrectly, via the command line or using the test yarn test runner.

Reply:
Ohhh. Does that fix the issue you were having which required you to switch to insiders?

Reply:
Yep!